miner/worker.go

```diff
@@ -962,7 +969,14 @@ func (w *worker) commitTransaction(tx *types.Transaction, coinbase common.Addres
 	return receipt.Logs, traces, nil
 }
 
-func (w *worker) commitTransactions(txs types.OrderedTransactionSet, coinbase common.Address, interrupt *int32) (bool, bool) {
+func (w *worker) commitTransactions(txs types.OrderedTransactionSet, coinbase common.Address, interrupt *int32) (r1 bool, r2 bool) {
+	withTimer(l2CommitBlockTimer, func() {
```

Comment:
Would be more useful to time `commitTransaction` and see how its duration is distributed among the 3 main operations: `traceEnv.GetBlockTrace`, `circuitCapacityChecker.ApplyTransaction`, and `core.ApplyTransaction`.

What insight do you expect to gain from timing `commitTransactions`?

Also fyi `commitTransactions` might be called 3x per block (for L1 messages, for locals, and for remotes).